src/main/resources/test_files/runtime-spark/DataFrames/GroupbyClause3.iq

```diff
@@ -0,0 +1,8 @@
+for $i in json-file("./src/main/resources/queries/confusion_sample.json")
+let $guess := $i."guess", $target := $i."target"
+where $guess eq $target
+group by $target
+order by $target
+return {"Language": $target,
+        "Correct guesses": count($i)}
```

Comment:
ForLetGroup.iq renamed to this file. Result order updated. Please compare for sanity check